packages/widgets/src/lib/utils/ViewUtils.ts

```diff
@@ -128,6 +128,37 @@ export function getPipelineTask(modelPipeline: PipelineType): PipelineType {
 	return modelPipeline === "text2text-generation" ? "text-generation" : modelPipeline;
 }
 
+interface RunShortcutParams {
+	isLoading: boolean;
+	isDisabled: boolean;
+	onClick: () => void;
+}
+
+/**
+ * Svelte action that will call inference endpoint when a user hits cmd+Enter on a current html element
+ */
+export function runShortCut(
+	node: HTMLElement,
+	{ isLoading, isDisabled, onClick }: RunShortcutParams
+): { destroy: () => void } {
+	function onKeyDown(e: KeyboardEvent) {
+		if (isLoading || isDisabled) {
+			return;
+		}
```

Comment:
Wouldn't it be simpler to simply pass one boolean instead of two?

Reply:
handled in 17541348049799a5ef5a0a17e84e2d5a64e69a5d